### PR TITLE
[RETINA-235] Add Tool Fixes

### PR DESCRIPTION
**Pull request by Jbcampbe**
image
image

### CODE REVIEW

File: src/styles/form-inputs.scss [hunk 1]
```diff
@@ -60,7 +65,8 @@
     color: white;
 
     &::placeholder {
-      color: $background-light-gray;
+      // color: $background-light-gray;
```
Comment (ekraihan) [inline src/styles/form-inputs.scss]: commented out code

Comment (ekraihan) [reply src/styles/form-inputs.scss]: 👃 
♨️

File: src/styles/form-inputs.scss [hunk 2]
```diff
@@ -36,7 +41,7 @@
   outline: none;
 
   &::placeholder {
-    color: $renascent-dark-gray;
+    color: #ACACAC;
```
Comment (ekraihan) [inline src/styles/form-inputs.scss]: Why duplicate this value multiple places?

**Comment by ekraihan** [comment]
Some notes on the add tool workflow:
- I can type jibberish into the model year field, move on, and it doesn't complain (it just clears the field)
- When I type "11/4/2018", as suggested, into the purchase-date field, the field clears as soon as I type a `/`. (And just as a note, I *am* able to type 11-4-2018 and it accepts it, and normalizes it to 11/4/2018)
- I can type jibberish into the price field, move on, and it doesn't complain (it just clears the field)
- When I type a non-existant purchased-from, the dropdown says "Set Type To <xxx>" versus "Set Purchased From To <xxx>" (copy paste mistake maybe?)

**Comment by amenders** [comment]
That is very strange.
Several of these are likely regressions.

-Amos Endersen

**Comment by Jbcampbe** [comment]
> Some notes on the add tool workflow:
> 
> * I can type jibberish into the model year field, move on, and it doesn't complain (it just clears the field)
> * When I type "11/4/2018", as suggested, into the purchase-date field, the field clears as soon as I type a `/`. (And just as a note, I _am_ able to type 11-4-2018 and it accepts it, and normalizes it to 11/4/2018)
> * I can type jibberish into the price field, move on, and it doesn't complain (it just clears the field)
> * When I type a non-existant purchased-from, the dropdown says "Set Type To " versus "Set Purchased From To " (copy paste mistake maybe?)

Fixed 2 and 4. 1 and 3 are a bit tricky. I decided to leave them as is (in order to prevent damaging the mobile UX) since nothing bad happens

**Pull request merged**